common/holders/receiptsHolder.go

```diff
@@ -0,0 +1,24 @@
+package holders
+
+import (
+	"github.com/ElrondNetwork/elrond-go-core/data/block"
+)
+
+type receiptsHolder struct {
```

Comment:
some unit tests for this components would have been great

Reply:
Added.